staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -231,6 +238,37 @@ func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionK
 		return list, nil
 	}
 
+	// We need to return resourceVersion for the List
+	// It should be the highest resourceVersion in etcd for the resource
+	// but given we know about only few objects picking the highest RV
+	// in the store is enough.
+	maxRv := uint64(1)
+	for _, obj := range objs {
+		rv, err := meta.NewAccessor().ResourceVersion(obj)
+		if err != nil {
+			return nil, err
+		}
+
+		if rv == "" {
+			// It's fine for unit tests
+			continue
+		}
+
+		// We are simulating the server so we are allowed to interpret RV
+		rvInt, err := strconv.ParseUint(rv, 10, 64)
+		if err != nil {
+			panic(fmt.Errorf("failed to parse resourceVersion %q", rv))
```

Comment:
bold, but I guess it's unlikely that someone using a mock client mocked this to a non-int

Reply:
I think this is fine, even a panic instead of error.

Reply:
changed it to an error so tests can continue to run and don't fail hard on the first offender